### DISCUSSION TITLE
How To change url in the canvas-ui

### DISCUSSION THREAD

**Discussion by premy-punchu** [austintoddj/canvas#1398] [General]
I want to change /canvas-ui how will I be able to do so

**Comment by austintoddj** [comment]
When you ran the `canvas-ui` Artisan command, the [applicable routes stubbed out here](https://github.com/austintoddj/canvas/blob/master/resources/stubs/routes/web.stub) were written to your base `routes/web.php` file. You can customize the URL however you'd like.

**Reply by premy-punchu** [reply]
Solved

**Reply by premy-punchu** [reply]
The url  have changed but after publishing post no post shown in the url

**Reply by austintoddj** [reply]
I'm not sure what you mean by that.
